osu.Game.Rulesets.Catch/UI/CatchPlayfieldAdjustmentContainer.cs

```diff
@@ -49,10 +52,13 @@ protected override void Update()
                 // in stable, fruit fall vertically from -100 to 340.
                 // to emulate this, we want to make our playfield 440 gameplay pixels high.
                 // we then offset it -100 vertically in the position set below.
-                const float stable_v_offset_ratio = 440 / 384f;
+                const float stable_v_offset_ratio = 440 / CatchPlayfield.HEIGHT;
+
+                // there is no explanation to this offset, but applying it brings lazer nearly 1:1 with stable.
+                const float stable_offset = 6f;
```

Comment:
May have something to do with https://github.com/ppy/osu/pull/9763. That said I can't find the original offset in stable source so may be a red herring.

Reply:
Question is, if this is the same thing then which out of `6` and `8` are more correct. I highly doubt it differs per ruleset.

Reply:
Hmm, you may be onto something. I'll look into this further more.

Reply:
"6" was apparently ballparked. So I'm betting on the fact that one (or both?) values were ballparked.

Reply:
Turns out the vertical offset that is applied in osu! ruleset is specific to it, stable does not apply any similar adjustment to catch.